### PR TITLE
Pam 326 sg quick start

### LINKED ISSUE #326: Style Guide Quick Start
Add a topic with the basic steps for getting started with style guides.

### PR DESCRIPTION
**Pull request by pamgoodrich**
Fixes #326.

Added two use cases: using shareable style guides and using local style guides.

### CODE REVIEW

File: 2a.-style-guides/style-guide-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Style Guide Quickstart
+
+Follow these steps to quickly use style guides that bring consistency and easy governance to your API program.
+
+You can take one of two approaches:
+
+* Create a style guide that you can shareacross all projects in your workspace. (Professional plan and above). See [Create a Shareable Style Guide](#create-a-shareable-style-guide).
```
Comment (heitortsergent) [inline 2a.-style-guides/style-guide-quick-start.md:7]: Small typo, shareacross

File: 2a.-style-guides/style-guide-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* Create rules in an API project. These rules cannot be shared. (Free plan and above.) See [Use a Local Style Guide](#use-a-local-style-guide).
```
Comment (heitortsergent) [inline 2a.-style-guides/style-guide-quick-start.md:8]: Can we remove the "(Free plan and above.)" part if it's available for all plans?

File: 2a.-style-guides/style-guide-quick-start.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Rules and targets are organized under the **Inherited** list in the left navigation panel. 
+
+1. Select an inherited rule.
+2. Review the rule's documentation. 
+3. Review the rule's properties.
+
+![style-guide-quick-start-sample-rule.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/I2UDPwPEW88)
+
+> If you do not want your API validated against a specific rule, select **Disabled** from the **Severity** list. ![Disable a rule](../assets/images/style-guide-disable-rule.png)
```
Comment (heitortsergent) [inline 2a.-style-guides/style-guide-quick-start.md:32]: Maybe we can add some clarification at the end of where the user can find the Severity field?

"... from the **Severity** list on the Edit pane."

File: 2a.-style-guides/style-guide-quick-start.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+>You can also [disable the entire default style guide](d-enable-style-guide.md#disable-a-style-guide).
+
+### 3. Create a Local Rule
+
+1. From the left pane, select the **Add** icon (+) next to **Rules**.
```
Comment (heitortsergent) [inline 2a.-style-guides/style-guide-quick-start.md:37]: @pamgoodrich what do you think about walking through creating an example rule here?

File: 2a.-style-guides/style-guide-quick-start.md (hunk 1 above)
Comment (pamgoodrich) [reply 2a.-style-guides/style-guide-quick-start.md:7]: Fixed. Thank you.

File: 2a.-style-guides/style-guide-quick-start.md (hunk 2 above)
Comment (pamgoodrich) [reply 2a.-style-guides/style-guide-quick-start.md:8]: Done

File: 2a.-style-guides/style-guide-quick-start.md (hunk 4 above)
Comment (pamgoodrich) [reply 2a.-style-guides/style-guide-quick-start.md:37]: Added. There's also a screenshot.

Comment (pamgoodrich) [reply 2a.-style-guides/style-guide-quick-start.md:37]: I'd like to see how it goes with the links to the examples for now.

**Review by mnaumanali94 — APPROVED** [review]

**Pull request merged**